assets/app.js

```diff
@@ -338,6 +338,39 @@ new Vue({
         }
     },
     computed: {
+        validateCacheMaterial: function () {
+            active_languages = this.activeLanguagesId
```

Comment:
when you use it as a normal method, do it for ALL languages

Reply:
Why?